specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```
@@ -2128,6 +2128,278 @@
           }
         }
       }
+    },
+    "/snapshots": {
+      "post": {
+        "description": "Submit an operation to take a snapshot of face list, large face list, person group or large person group, with user-specified snapshot type, source object id, apply scope and an optional user data.<br/><br/>The snapshot interfaces are for users to backup and restore their face data from one face subscription to another, inside same region or across regions. The workflow contains two phases, user first calls Snapshot - Take to create a copy of the source object and store it as a snapshot, then calls Snapshot - Apply to paste the snapshot to target subscription. The snapshots are stored in a centralized location (per Azure instance), so that they can be applied cross accounts and regions.<br/><br/>Taking snapshot is an asynchronous operation. An operation id can be obtained from the \"Operation-Location\" field in response header, to be used in OperationStatus - Get for tracking the progress of creating the snapshot. The snapshot id will be included in the \"resourceLocation\" field in OperationStatus - Get response when the operation status is \"succeeded\".<br/><br/>Snapshot taking time depends on the number of person and face entries in the source object. It could be in seconds, or up to several hours for 1,000,000 persons with multiple faces.<br/><br/>Snapshots will be automatically expired and cleaned in 48 hours after it is created by Snapshot - Create. User can delete the snapshot using Snapshot - Delete by themselves any time before expiration.<br/><br/>Taking snapshot for a certain object will not block any other operations against the object. All read-only operations (Get/List and Identify/FindSimilar/Verify) can be conducted as usual. For all writable operations, including Add/Update/Delete the source object or its persons/faces and Train, they are not blocked but not recommended because writable updates may not be reflected on the snapshot during its taking. After snapshot taking is completed, all readable and writable operations can work as normal. Snapshot will also include the training results of the source object, which means target subscription the snapshot applied to does not need re-train the target object before calling Identify/FindSimilar.",
```

```suggestion
Snapshots will be automatically expired and cleaned in 48 hours after it is created by Snapshot - Create.
```
It should be Snapshot - Take.

---

Done, thanks.